python/pyrogue/_Command.py

```diff
@@ -219,52 +219,10 @@ def get(self, read=True):
 
         return ret
             
-# Legacy Support
-def Command(offset=None, **kwargs):
-    if offset is None:
-
-        # Get list of possible class args
-        cargs = inspect.getfullargspec(LocalCommand.__init__).args + \
-                inspect.getfullargspec(LocalCommand.__init__).kwonlyargs
-
-        # Pass supported args
-        args = {k:kwargs[k] for k in kwargs if k in cargs}
-
-        ret = LocalCommand(**args)
-        ret._depWarn = True
-        return(ret)
-    else:
-
-        # Get list of possible class args
-        cargs = inspect.getfullargspec(RemoteCommand.__init__).args + \
-                inspect.getfullargspec(RemoteCommand.__init__).kwonlyargs
-
-        # Pass supported args
-        args = {k:kwargs[k] for k in kwargs if k in cargs}
-
-        ret = RemoteCommand(offset=offset,**args)
-        ret._depWarn = True
-        return(ret)
-
 Command.nothing = BaseCommand.nothing
```

Comment:
Are these still necessary? Maybe we should just find any code affected and refactor it.

Reply:
That section go removed in the PR I thought?